include/grpc/impl/codegen/port_platform.h

```diff
@@ -114,6 +114,38 @@
 #define GPR_WIN32_ATOMIC 1
 #define GPR_MSVC_TLS 1
 #endif
+#elif defined(GPR_MANYLINUX1)
+// TODO(atash): manylinux1 is just another __linux__ but with ancient
+// libraries; it should be integrated with the `__linux__` definitions below.
```

Comment:
Re: the comment: do you have a "proper" way to detect "manylinux" from within __linux__ ? And in fact, a good portion of these could be folded inside __linux__ using GPR_MANYLINUX1 as you just said - it just needs to be detected, either manually using GPR_MANYLINUX1, or any other definition that your compiler would do. Either way, this is fine.